components/task-management/TaskAnalyticsChart.tsx

```
@@ -0,0 +1,664 @@
+/**
+ * Get task type color
+ */
+function getTaskTypeColor(taskType: TaskType, isDark: boolean): string {
+  const colors = TASK_TYPE_COLORS[taskType as keyof typeof TASK_TYPE_COLORS];
+  return colors ? (isDark ? colors.dark : colors.light) : (isDark ? '#60A5FA' : '#3B82F6');
+}
+
+/**
+ * Render chart based on type
+ */
+const renderChart = useCallback((chartData: any, screenWidth: number) => {
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)